agentos/registry.py

```diff
@@ -226,13 +274,35 @@ def get_component_specs(
                 return {}
         return self._registry["components"]
 
-    def get_repo_spec(self, repo_id: str) -> "RepoSpec":
-        return self._registry["repos"][repo_id]
-
-    def get_run_spec(self, run_id: str) -> Dict:
-        return self._registry["runs"][run_id]
-
-    def get_registries(self) -> List[Registry]:
+    def get_repo_spec(self, repo_id: str, flatten: bool = False) -> "RepoSpec":
```

Comment:
it looks like the `get_*_spec()` functions are all pretty much the same.  Would it make sense to dedup the functionality?

Reply:
Done.